OpenEdXMobile/src/main/java/org/edx/mobile/module/analytics/FirebaseAnalytics.java

```diff
@@ -739,4 +739,13 @@ public void trackCourseCelebrationShareClicked(@NonNull String courseId, @Nullab
         }
         logFirebaseEvent(event.getName(), event.getBundle());
     }
+
+    @Override
+    public void trackResumeCourseBannerTapped(@NonNull String courseId, @NonNull String blockId) {
+        final FirebaseEvent event = new FirebaseEvent(Events.RESUME_COURSE_BANNER, Values.RESUME_COURSE_BANNER_TAPPED);
```

Comment:
The event is should be `Resume Course Tapped` as per the story.

Reply:
Done